venues/AKBC.ws/2019/Conference/python/invitations.py

```diff
@@ -16,49 +16,20 @@
 import os
 import time
 
-archival_status = {
-    'archival_status': {
-        'order': 1,
-        'value-dropdown': {},
-        'description': 'Update archival status.',
-        'required': True
-    }
-}
-
-archival_status_template = {
-    'id': conferenceConfig.ARCHIVAL_STATUS_TEMPLATE_STR,
+change_archival_status_template = {
+    'id': conferenceConfig.CONFERENCE_ID + '/-/Paper<number>/Change_Archival_Status',
```

Comment:
I'm interested in other opinions, but my thought about naming invitations is that they should be named for nouns, not verbs. So instead of "Change_Archival_Status", maybe it should just be "Archival_Status"?

@melisabok what do you think?

Reply:
I like using nouns, 'Archival Status' is better. 

I know that we have 'Add Revision', we should rename it in the future.

Reply:
ah yeah, it should probably just be "Revision"

Reply:
good point, but I feel that a user would find "Add Revision" and "Change Archival Status" more instructive than "Revision" and "Archival Status". What do you guys think ?